api/delete-sweep.md

```diff
@@ -0,0 +1,67 @@
+Deleting All Data Before A Date
+==================================
+
+This API call is for deleting any type of data from the IRONdb cluster prior to
+a given date, irrespective of check or metric. This is typically used for
+cluster-wide retention policies.
+
+This call will return an empty array upon success. If there is an error, this
+call will return a JSON object with the error.
+
+Description of API call
+-----------------------
+
+**URI:**   /sweep\_delete
+
+**Method:**   DELETE
+
+**Inputs:**   
+
+None. All options are provided by Headers (see below).
+
+**Headers:**
+
+The timestamp is provided via a header:
+
+X-Snowth-Delete-Time: &lt;new\_epoch&gt;
+
+The type of data (numeric, text, histogram) to remove may be specified, with
+multiple values comma-separated. If this header is not specified, only numeric
+data (nnt) will be removed.
+
+X-Snowth-Delete-Data-Types: nnt,text,histogram
+
+Specific NNT rollup spans may be removed (all spans will be removed if not
+specified), with multiple values comma-separated.
```

Comment:
These spans should match the spans configured in your `irondb.conf` file under `//rollups/rollup`